lib/client.js

```diff
@@ -116,23 +116,30 @@ SearchSource.prototype.getData = function(options) {
     selector = {};
   }
 
-
   function transform(doc) {
-    self.searchFields.forEach(function(field) {
-      if(self.currentQuery && doc[field]) {
-        if(options.transform) {
+    if(options.transform) {
+      self.searchFields.forEach(function(field) {
+        if(self.currentQuery && doc[field]) {
           doc[field] = options.transform(doc[field], regExp, field, self.currentQuery);
         }
-      }
-    });
+      });
+    }
+    if(options.docTransform)
```

Comment:
Use brace style: https://github.com/meteorhacks/code-standards/blob/master/docs/style_guide.md#brace-style

follow that for others as well.